src/os/bluestore/BlueFS.cc

```diff
@@ -104,16 +104,16 @@ class BlueFS::SocketHook : public AdminSocketHook {
 	   Formatter *f,
 	   std::ostream& errss,
 	   bufferlist& out) override {
-    if (command == "bluestore bluefs available") {
+    if (command == "bluestore bluefs device info") {
       int64_t alloc_size = 0;
```

Comment:
I like change "available" -> "device info".
You missed to change this also at `register_command`.

Reply:
yep, my bad. Fixed now. Thanks!